src/components/About/Blog/BlogPost.js

```diff
@@ -0,0 +1,339 @@
+import React from 'react';
+import _Card from '@material-ui/core/Card';
+import CardMedia from '@material-ui/core/CardMedia';
+import CardContent from '@material-ui/core/CardContent';
+import _Typography from '@material-ui/core/Typography';
+import Button from '@material-ui/core/Button';
+import {
+  FacebookShareButton,
+  TwitterShareButton,
+  FacebookIcon,
+  TwitterIcon,
+} from 'react-share';
+import styled, { css } from 'styled-components';
+import Fade from '@material-ui/core/Fade';
+import './Blog.css';
+import dateFormat from 'dateformat';
+import htmlToText from 'html-to-text';
+import renderHTML from 'react-render-html';
+import susi from '../../../images/susi-logo.svg';
+import PropTypes from 'prop-types';
+
+const allCategories = [
+  'FOSSASIA',
+  'GSoC',
+  'SUSI.AI',
+  'Tutorial',
+  'Android',
+  'API',
+  'App generator',
+  'CodeHeat',
+  'Community',
+  'Event',
+  'Event Management',
+  'loklak',
+  'Meilix',
+  'Open Event',
+  'Phimpme',
+  'Pocket Science Lab',
+  'yaydoc',
+];
+
+const arrDiff = (a1, a2) => {
```

Comment:
Are you trying to intersect 2 arrays here?

Check out lodash operations and use directly.